src/main/java/com/laserfiche/repository/api/clients/impl/model/APIServerException.java

```diff
@@ -2,26 +2,175 @@
+    }
+
+    public APIServerException errorCode(Integer errorCode) {
+        this.errorCode = errorCode;
+        return this;
+    }
+
+    @Schema(description = "")
+    public Integer getErrorCode() {
+        return errorCode;
+    }
+
+    public void setErrorCode(Integer errorCode) {
+        this.errorCode = errorCode;
+    }
+
+    public APIServerException errorClass(String errorClass) {
+        this.errorClass = errorClass;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getErrorClass() {
+        return errorClass;
+    }
+
+    public void setErrorClass(String errorClass) {
+        this.errorClass = errorClass;
+    }
+
+    public APIServerException statusCode(Integer statusCode) {
+        this.statusCode = statusCode;
+        return this;
+    }
+
+    @Schema(description = "")
+    public Integer getStatusCode() {
+        return statusCode;
+    }
+
+    public void setStatusCode(Integer statusCode) {
+        this.statusCode = statusCode;
+    }
+
+    public APIServerException errorSource(String errorSource) {
+        this.errorSource = errorSource;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getErrorSource() {
+        return errorSource;
+    }
+
+    public void setErrorSource(String errorSource) {
+        this.errorSource = errorSource;
+    }
+
+    @Override
+    public boolean equals(java.lang.Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        APIServerException apIServerException = (APIServerException) o;
+        return Objects.equals(this.operationId, apIServerException.operationId) && Objects.equals(this.message,
+                apIServerException.message) && Objects.equals(this.errorCode,
+                apIServerException.errorCode) && Objects.equals(this.errorClass,
+                apIServerException.errorClass) && Objects.equals(this.statusCode,
+                apIServerException.statusCode) && Objects.equals(this.errorSource, apIServerException.errorSource);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(operationId, message, errorCode, errorClass, statusCode, errorSource);
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("class APIServerException {\n");
+        sb
+                .append("    operationId: ")
+                .append(toIndentedString(operationId))
+                .append("\n");
+        sb
+                .append("    message: ")
+                .append(toIndentedString(message))
+                .append("\n");
+        sb
+                .append("    errorCode: ")
+                .append(toIndentedString(errorCode))
+                .append("\n");
+        sb
+                .append("    errorClass: ")
+                .append(toIndentedString(errorClass))
```

Comment:
Maybe it's better to update the code generator to generate a more compact code like
sb.append(String.format("    errorClass: %s%n"), toIndentedString(errorClass));

or 

sb.append(String.format("    errorClass: %s%n"), String.valueOf(errorClass));

This also removes the toIndentedString which currently is duplicated in all the classes.

Reply:
I agree. I think it could be a future story (the current one works).